draft-ietf-quic-http.md

```diff
@@ -105,6 +105,11 @@ Connectivity problems (e.g. firewall blocking UDP) can result in QUIC connection
 establishment failure, in which case the client SHOULD continue using the
 existing connection or try another alternative endpoint offered by the origin.
 
+Servers MAY serve HTTP/QUIC on any UDP port.  Servers MUST use the same port
+across all IP addresses that serve a single domain, and SHOULD NOT change this
+port. Servers SHOULD choose this port at random when first enabling HTTP/QUIC
```

Comment:
I'd just remove it -- if it's just a MAY, then it's duplicative of "MAY serve ... on any UDP port."